### ISSUE TITLE
Elementor tabs issue

### ISSUE THREAD

**Issue by Demoka** [pojome/elementor#4080]
hello, i have a problem with tabs widget in elementor.
i need to make two links which are going on a page with two tabs
the first link may open this page on a first tab, second link need to open this page on a second tab.
![30442503_1463436413783721_492687174676774912_n](https://user-images.githubusercontent.com/35471046/38670946-d4e90b0c-3e52-11e8-8fb5-2741628e3c78.png)
i add screenshot to provide problem visualization.

**Comment by bainternet** [comment]
Currently, this feature is not supported, but we will look into it in the future if we see a large demand for it.

What you can do for now is with some minor custom coding, on the page load check if you have a value in the `document.location.hash` and if you do just set the focus to it which in turn will trigger the change of the tab. something like:

```JS
jQuery( document ).ready( function($){	
	if ( window.location.hash ) {
		try{ 
			$( window.location.hash ).focus().focus()
		}catch( err ) {}
	}
})
```

And make sure to set the links to the same page but each with the hash of the id of the tab-title, in your case it would be `http://belogub.net/ico/#elementor-tab-title-8892`
Thanks for the feedback.

**Comment by cyrilmesvayn** [comment]
Does this workaround still supposed to work as of 2.3.1 ?

I've tried that either with .focus().focus() or .trigger("focus") and it doesn't seems to work anymore [Screen capture available](https://imgur.com/sWuNwDr).

This is made from a fresh WP 4.9.8 install and Elementor 2.3.1.

Or is this any programmatic way to access Tabs Widget instance and externally call changeActiveTab ?

**Comment by sebinievas** [comment]
@bainternet @cyrilmesvayn did you ever get this figured out?

**Comment by ComputerTinker** [comment]
For those finding this page after searching for changeActiveTab, I was not able to determine a good way to invoke that function externally.  The simplest solution I've found is just to manually trigger the click event on the desired tab title.

**Comment by JoeSnuffie** [comment]
Late to the game here but I struggled with this as well. My issue seems to be related to a plugin or theme. I have a plugin that modifies the theme and several plugins we use and activating the tab is a must for us.  The DOM click events were firing but for unknown reasons the tab wouldn't activate. I added a button that fired the same event and it would work when I clicked it. I'm assuming that the preloader or some other code was causing the page to not be ready even though the console showed the DOM clicks I was firing. By the way, in my case only DOM clicks would work. jQuery click didn't work, although focus did but that won't change the tab.

I enqueue in my main plugin file
`wp_enqueue_script('my-plugin-elementor', plugin_dir_url( __DIR__ ) . 'my-plugin/js/elementor.js','','',true);`
I pass the tab I want activated in the URL like ?tab=elementor-tab-title-3363
This is my JS
`jQuery(window).on('load', function () {
	const params = new Proxy(new URLSearchParams(window.location.search), {
      get: (searchParams, prop) => searchParams.get(prop),
    });
    let tab = params.tab; 

    if (tab != ""){
        document.getElementById(tab).click();
    }
});`
